ls1intum/Artemis
# `General`: Add information for manual LTI 1.3 configuration

### Checklist
#### General
#### Client
### Motivation and Context

It is possible to dynamically register an LTI course as a tool in other Learning Management Systems. However, in Moodle, to use the dynamic registration it is necessary to be an administrator of the system. Therefore, it might be necessary in this and other cases to manually register an Artemis course in a platform. To do this, it is necessary to show the instructor a few more configuration values in the LTI Configuration page.

### Description

Adds/shows the necessary configuration values in the **course-lti-configuration-page** component

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to a course
4. Enable the online course
5. Navigate to the LTI configuration page
6. Go to the LTI 1.3 tab
7. Verify the new "Tool URL", "Initiate Login URL" and "Redirect URI" fields are present, have the correct values, and it is possible to copy them
8. Verify the english/german translations are present and are the same (as it doesn't make sense to translate them)

### Review Progress

#### Code Review
#### Manual Tests

### Test Coverage
| Class/File | Branch | Line |
|------------|-------:|-----:|
| course-lti-configuration.component.ts | 75% | 100% |

### Screenshots

New fields in LTI 1.3 tab: 
![image](https://user-images.githubusercontent.com/11006274/217564795-87f0d71c-4bb0-4887-b7d6-d606276f2e86.png)
